webapps/frontend/app/viewmodels/site/admin/support.js

```diff
@@ -109,23 +121,59 @@ define([
             dialog.show(wizard);
             return true;  // Critical to return True, otherwise knockout can't handle a click + checked event
         };
+        self.configureGraphite = function(edit) {
+            var showWizard = false;
+            if (edit === true) {
+                showWizard = true;
+            } else {
+                if (!self.selectedSupportSettings().contains('fwk_statistics') && self.origGraphiteSettings.isInitialized() === false) {
+                    showWizard = true;
+                }
+            }
+
+            if (showWizard === false) {
+                return true;  // Critical to return True, because returning true makes sure that the default 'click' handler gets triggered too (required for the 'checked' handler
+            }
+            var wizard = new GraphiteConfigureWizard({
+                modal: true,
+                newConfig: self.newGraphiteSettings,
+                origConfig: new GraphiteSettingsVM(self.origGraphiteSettings.toJS())
+            });
+            wizard.closing.always(function() {
+                if (self.origGraphiteSettings.isInitialized() === false) {
+                    var index = self.selectedSupportSettings.indexOf('fwk_statistics');
+                    if (index > -1) {
+                        self.selectedSupportSettings.splice(index, 1);
+                    }
+                }
+            });
+            wizard.finishing.always(function() {
+                if (self.supportSettingsChanged() === true) {
+                    self.save();
+                }
+            });
+            dialog.show(wizard);
+            return true;  // Critical to return True, otherwise knockout can't handle a click + checked event
```

Comment:
Returning True allows your browser to run the default click behavior. This comment makes it look like knockout is incompetent at it